### ISSUE TITLE
[Suggestion] More then only one search language

### ISSUE THREAD

**Issue by GambaJo** [searx/searx#1344]
I have to choose a language for the search. Thst's ok. But often this is not enough.
In other search engines, like duckduckgo or google I can choose the search language too, but they show me also some english results.

I'm a software developer and I often search for some code or solutions. I want to search in my language (german) but often the solutions I'm looking for are in english. In other search engines I have both. In searx I have to switch the language everytime I want the english results too.

**Comment by asciimoo** [comment]
@GambaJo you can specify the language in the search query, too: https://asciimoo.github.io/searx/user/search_syntax.html .

**Comment by GambaJo** [comment]
@asciimoo You did't read my text. Thx for nothing.

**Comment by Pofilo** [comment]
What @GambaJo means is that if he makes a research in a language (let's say french), he will only get results from this language.
As a developer, a lot of results came in english on google/ddg .. So he doesn't get them with searx and is forced to make a new research specifying english.

**Comment by GambaJo** [comment]
@Pofilo This is exactly, what I ment. Thx!

**Comment by thlautenschlaeger** [comment]
same issue for me. would be cool to have saved fast switching

**Comment by ilf** [comment]
I second the request for an option to specify multiple languages. Something like `https://searx.xyz/?q=test&language=de,en` would be ideal.

**Comment by oyren** [comment]
This prevents me from using searx.

**Comment by ghost** [comment]
Same thing, I want but can't use searx because of it.

**Comment by ghost** [comment]
I also frequently just use another search engine, because searx started to filter search results to show only english results (talk about filter bubbles…). This is fine for people who speak only English, but I would like to see German and Polish, too, for instance. However, the user should not have to "unfilter" English; the default case should be that there is *no* language filter at all (with the option to exclude or prioritise languages).

In short: don't filter prematurely; let the user filter, not the server.

**Comment by miicha** [comment]
For me, it would be sufficient, if I can search in english and one other selected language. Therefore I implemented kind of a quick and dirty solution: https://github.com/asciimoo/searx/pull/1419.

It can be tested at: https://finden.tk/

**Comment by GambaJo** [comment]
@miicha Looks good to me (compared to duckduckgo)
![duckduckgo](https://user-images.githubusercontent.com/1378907/47215752-d5cdd800-d3a2-11e8-9ff5-78ab40867efb.jpg)
![searx](https://user-images.githubusercontent.com/1378907/47215753-d5cdd800-d3a2-11e8-89e1-652670bf9d43.jpg)

**Comment by ilf** [comment]
@miicha Thanks, that's a great start!

But that one *always* searches in `en_US` *plus* the defined language in `language=`? That's okay for a first proof-of-concept, but really an ideal solution would allow specifying *any* two languages - *and* specify them in `language=` for links and browser-plugins etc.

**Comment by bruvv** [comment]
Since this is pretty old and still an issue or I have it mis configured... But what about multi language support?

**Comment by danilovicente** [comment]
I don't know for you guys, I'm from Brazil, but when I leave on Default language if I type something in portuguese that return portuguese sites for me, when I search something in english that return english results for me.

**Comment by dosch** [comment]
I still have this issue. I am 0.18.0

**Comment by dev-noah** [comment]
I have the same issue on the latest version, it is the only thing holding me back from always using searx.
